### DISCUSSION TITLE
Can I use Ergo in production and can you provide us with some successful projects using Ergo?

### DISCUSSION THREAD

**Discussion by leonlee2013** [ergo-services/ergo#131] [Q&A] [answered]
Ergo is a great idea, very easy to integrate Erlang and form an Erlang/Go cluster, taking full advantage of Go's high performance and Erlang's features.

We are currently testing and investigating whether Ergo can be used in a production environment, can you provide us with some success stories?

**Comment by halturin** [comment, marked as answer]
It's already used in production, even in high-load projects (DPI solution from https://ringcentral), and in heavy data-processing projects (TransparentDeal from Kaspersky Lab https://kaspersky.com). It's also used in some projects which I can't expose due to NDA.

I can also add your company name on https://ergo.services if you decide to use this framework in production (if you don't mind, for sure)

For commercial support, please get in touch with ceo@ergo.services

**Reply by leonlee2013** [reply]
Thank you so much for your reply!

**Comment by rupurt** [comment]
@leonlee2013 I'm also using it in production for trading and will be open sourcing much of the framework. It builds upon the Elixir trading libraries that I've already authored e.g. https://github.com/fremantle-industries/tai
